plugin/output/s3/s3.go

```diff
@@ -74,13 +82,28 @@ type Config struct {
 	Endpoint  string `json:"endpoint" required:"true"`
 	AccessKey string `json:"access_key" required:"true"`
 	SecretKey string `json:"secret_key" required:"true"`
-	Bucket    string `json:"bucket" required:"true"`
-	Secure    bool   `json:"secure" default:"false"`
+	// Required s3 default bucket.
+	Bucket string `json:"bucket" required:"true"`
```

Comment:
let's rename to default_bucket

Reply:
Let's remain in field to DefaultBucket and keep json `bucket` to not break usage.